pybikes/samba.py

```diff
@@ -114,3 +114,59 @@ def update(self, scraper=None):
                 'status': self.get_status(online_status, operation_status)
             }
             self.stations.append(station)
+
+class SambaArgentina(BaseSystem):
+    sync = True
+    _STATIONS_RGX = "var\ estacoes\ =\ (\[.*\]);//]]>"
+
+    def __init__(self, tag, url, meta):
+        super(SambaArgentina, self).__init__(tag, meta)
+        self.feed_url = url
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+
+        html_data = scraper.request(self.feed_url)
+        string_data = re.search(SambaArgentina._STATIONS_RGX, html_data).group(1)
+        strs = string_data.replace('[','').split('],')
+        data = [map(str, s.encode('utf-8').replace(']','').split(',')) for s in strs]
+
+        # Each station is like follows
+        # [
+        #   'Pellegrini',
+        #   '-32.953886',
+        #   '-60.656628',
+        #   '',
+        #   'A',
+        #   'EO',
+        #   '9',
+        #   '9',
+        #   '11',
+        #   'Est_Normal 1',
+        #   './img/ico_operacao.png',
+        #   '1',
+        #   'Museo Castagnino',
+        #   'en operación',
+        # ]
+
+        stations = []
+
+        for item in data:
+            name = item[12].replace("'", "")
+            latitude = float(item[1].replace("'", ""))
+            longitude = float(item[2].replace("'", ""))
+            bikes = int(item[7].replace("'", ""))
+            free = int(item[8].replace("'", ""))
+            total_slots = free + bikes
+            extra = {
+                'slots': total_slots,
+                'dirección': item[0].replace("'", ""),
+                'uid': item[11].replace("'", ""),
+                'status': item[13].replace("'", ""),
+            }
+            station = BikeShareStation(name, latitude, longitude, bikes, free,
+                                       extra)
+            stations.append(station)
+
+        self.stations = stations
```

Comment:
We recently included `demjson` which correctly parses JS type objects.

Following works:

```python
import demjson

raw_data = re.search(r'var estacoes = (\[.+\]);', html_data).group(1)
data = demjson.deocde(raw_data)
>>> print data
[[u'Pellegrini', u'-32.953886', u'-60.656628', '', u'A', u'EO', u'10', u'10', u'10', u'Est_Normal 1', u'./img/ico_operacao.png', u'1', u'Museo Castagnino', u'en operaci\xf3n'], [u'Pellegrini', u'-32.956960', u'-60.644427', '', u'A', u'EO', u'14', u'14', u'6', u'Est_Normal 1', u'./img/ico_operacao.png', u'2', u'Paseo Pellegrini', u'en operaci\xf3n'], [u'Pellegrini ', u'-32.958351', u'-60.636239', '', u'A', u'EO', u'13', u'13', u'7', u'Est_Normal 1', u'./img/ico_operacao.png', u'3', u'Plaza L\xf3pez', u'en operaci\xf3n'], [u'Pellegrini', u'-32.959327', u'-60.628885', '', u'A', u'EO', u'15', u'15', u'5', u'Est_Normal 1', u'./img/ico_operacao.png', u'4', u'Fac. de Ingenier\xeda', u'en operaci\xf3n'], [u'Rosario', u'-32.965547', u'-60.623939', '', u'A', u'EO', u'0', u'0', u'21', u'Est_Vazia -1', u'./img/ico_sem_bike.png', u'5', u'Ciudad Univ. I', u'en operaci\xf3n'], [u'Rosario', u'-32.966281', u'-60.623568', '', u'A', u'EO', u'6', u'6', u'14', u'Est_Normal 1', u'./img/ico_operacao.png', u'6', u'Ciudad Univ. II', u'en operaci\xf3n'], [u'Cafferata', u'-32.940079', u'-60.670702', '', u'A', u'EO', u'8', u'8', u'13', u'Est_Normal 1', u'./img/ico_operacao.png', u'7', u'Terminal', u'en operaci\xf3n'], [u'Francia', u'-32.940056', u'-60.665159', '', u'A', u'EO', u'16', u'16', u'5', u'Est_Normal 1', u'./img/ico_operacao.png', u'8', u'Fac. Medicina', u'en operaci\xf3n'], [u'Pueyrred\xf3n', u'-32.945260', u'-60.658115', '', u'A', u'EO', u'10', u'10', u'2', u'Est_Normal 1', u'./img/ico_operacao.png', u'9', u'Pueyrred\xf3n', u'en operaci\xf3n'], [u'Bvrd. Oro\xf1o', u'-32.949595', u'-60.654339', '', u'A', u'EO', u'8', u'8', u'12', u'Est_Normal 1', u'./img/ico_operacao.png', u'10', u'Fac. Ciencias Econ\xf3micas', u'en operaci\xf3n'], [u'C\xf3rdoba', u'-32.943842', u'-60.650070', '', u'A', u'EO', u'6', u'6', u'14', u'Est_Normal 1', u'./img/ico_operacao.png', u'11', u'Plaza San Mart\xedn', u'en operaci\xf3n'], [u'C\xf3rdoba', u'-32.945053', u'-60.644409', '', u'A', u'EO', u'3', u'3', u'17', u'Est_Normal 1', u'./img/ico_operacao.png', u'12', u'Plaza Pringles', u'en operaci\xf3n'], [u'San Luis', u'-32.949294', u'-60.642499', '', u'A', u'EO', u'10', u'10', u'10', u'Est_Normal 1', u'./img/ico_operacao.png', u'13', u'Plaza Sarmiento', u'en operaci\xf3n'], [u'San Luis', u'-32.949528', u'-60.637714', '', u'A', u'EO', u'8', u'8', u'13', u'Est_Normal 1', u'./img/ico_operacao.png', u'14', u'Plaza Montenegro', u'en operaci\xf3n'], [u'Buenos Aires', u'-32.947039', u'-60.633235', '', u'A', u'EO', u'11', u'11', u'9', u'Est_Normal 1', u'./img/ico_operacao.png', u'15', u'Plaza 25 de Mayo', u'en operaci\xf3n'], [u'Salta', u'-32.936746', u'-60.651647', '', u'A', u'EO', u'13', u'13', u'7', u'Est_Normal 1', u'./img/ico_operacao.png', u'16', u'Pichincha', u'en operaci\xf3n'], [u'Wheelwright', u'-32.936670', u'-60.641238', '', u'A', u'EO', u'9', u'9', u'11', u'Est_Normal 1', u'./img/ico_operacao.png', u'17', u'Distrito Centro', u'en operaci\xf3n'], [u'Los Inmigrantes', u'-32.940916', u'-60.634859', '', u'A', u'EO', u'5', u'5', u'15', u'Est_Normal 1', u'./img/ico_operacao.png', u'18', u'Parque Espa\xf1a', u'en operaci\xf3n']]
```